code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -295,3 +295,22 @@
 	icon_state = "pill8"
 	list_reagents = list(/datum/reagent/iron = 30)
 	rename_with_volume = TRUE
+	
+/obj/item/reagent_containers/pill/floor_pill
+	name = "☼Floor Pill☼"
+	desc = "This is a masterful floor pill. It is decorated with ashwalker bone and encircled with bands of iron and silver. The object menaces with spikes of wood. On the item is an image of an assistant eating a floor pill. The assistant is dying. On the item is an image of four medical doctors. The medical doctors are praying."
+	icon_state = "pill8"
```

Comment:
```suggestion
	icon_state = "pill8"
	reagent_flags = NO_REACT
```

I don't think you want this rolling RDX + Teslium and instantly breaching maint at roundstart.